### ISSUE TITLE
Protocol Amendments [CT]

### ISSUE THREAD

**Issue by BSnoeijerCD** [cdisc-org/DDF-RA#474]
Need to ensure that M11 protocol amendments can be supported. Three aspects

1. Amendment support aligns with the updated M11 specification
2. Ensure we can support the detailed rationale for changes. Note we don't need exact details of change as an electronic comparison can be made (see #305)
3. Align with any review comments on Organizations arising from release 3.6. This overlaps because of the need to note enrollment at the time of an amendment being approved.

See feature ticket #469

**Comment by ASL-rmarshall** [comment]
@EMuhlbradt @czwickl (FYI @BSnoeijerCD )
I have reviewed the 469 spreadsheet posted [here](https://github.com/cdisc-org/DDF-RA/issues/469#issuecomment-2375172384) and have the following comments:
| Sheet  | Cell  | Object                                 | Column           | Cell Contents                                                                                                                                                      | Comment                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
| ------ | ----- | -------------------------------------- | ---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| new    | $G$3  | GovernanceDate.type                    | UML Item Name    | GovernanceDate.type                                                                                                                                                | It looks like this cell had been inserted shifting cells below downwards and misaligning the descriptions. My review was done after deleting this cells.                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
| new    | $K$2  | GovernanceDate.type                    | Preferred Term   | Study Version Approval Date                                                                                                                                        | Based on [Dave's comment](https://github.com/cdisc-org/DDF-RA/issues/469#issuecomment-2358126974), I think the intent may have been to create type terminology just for "Approval Date", "Effective Date", etc. with the context determined by where the governance date is used. "Issued Date" was an additional suggestion for consideration.                                                                                                                                                                                                                                                            |
| new    | $M$11 | StudyAmendment.changes                 | CDISC Definition | A USDM relationship between the StudyAmendment and  StudyAmendmentChange classes which identifies the set of changes associated with the study amendment.          | StudyAmendmentChange should be StudyChange                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
| new    | $M$22 | StudyChange.changedSections            | CDISC Definition | A USDM relationship between the StudyChange and DocumentContentReference classes which provides the set of changed document sections related to the study changes. | I think the final 's' should be removed - "related to the study change."                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
| new    | $M$23 | DocumentContentReference               | CDISC Definition | A citation pointing to the location of specific content within the document.                                                                                       | For the DocumentContentReference entity and all its attributes, I think "a document" might be better than "the document" in the descriptions because there is no direct association between the document content reference and any specific document                                                                                                                                                                                                                                                                                                                                                       |
| new    | $M$24 | DocumentContentReference.sectionNumber | CDISC Definition | The numeric identifier assigned to a particular section for the document content reference.                                                                        | For both sectionNumber and sectionTitle in DocumentContentReference, I think the descriptions may be largely based on those from NarrativeContent. However, in NarrativeContent the section number/title is being assigned, whereas in DocumentContentReference, an existing number/title is being referenced. I think the wording of the DocumentContentReference attribute should be modified to reflect this - e.g., something like "The numeric identifier of a particular section referenced within a document.", "The identifying designation of a particular section referenced within a document." |
| new    | $M$28 | StudyAmendmentImpact.isSubstantial     | CDISC Definition | An indication as to whether the study amendment has a substantial impact on some aspect of the study.                                                              | I think this should be "An indication as to whether the impact of the study amendment on some aspect of the study is substantial."                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
| new    | $M$31 | StudyAmendmentImpact.type              | CDISC Definition | The risk of harm or adverse effect by some aspect of the study that a study subject may present with.                                                              | Suggestion: An alteration by some aspect of the study to the risk of harm or adverse effect with which a study subject may be presented.                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
| new    | $M$32 | StudyAmendmentImpact.type              | CDISC Definition | The risk of xxxxx by some aspect of the study that a study subject may experience.                                                                                 | Suggestion: An alteration by some aspect of the study to the rights of study subjects.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
| new    | $K$33 | StudyAmendmentImpact.type              | Preferred Term   | Study Data Reliability or Robustness                                                                                                                               | Based on Craig's input, I think we agreed to separate reliabillity and robustness                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
| change | $G$4  | SubjectEnrollment                      | UML Item Name    | quantity                                                                                                                                                           | As discussed at the scrum today, SubjectEnrollment quantity remains unchanged.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
| change | $G$5  | SubjectEnrollment                      | UML Item Name    | type                                                                                                                                                               | As discussed at the scrum today, the type and code attributes inherited from GeographicScope should be removed from SubjectEnrollment                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |

**Comment by ASL-rmarshall** [comment]
Also, there's no mention in the Excel file of the removal of `substantialImpact` from `StudyAmendment`

**Comment by BSnoeijerCD** [comment]
@EMuhlbradt @czwickl Review notes (including review of comments Richard above).
- It was F3 which was inserted not G3 causing the rest of the column to be misaligned.
- Ticket of Dave indicates all the different kinds of dates per type as now implemented in the CT sheet. However. we can discuss whether the context needs to be included in the type given the date is always specified within a context. 
- Can we add issued and signature date types? Or is the Signature date seen as the approval date?
- If we specify a context, I would not add "Protocol" to "Protocol Amendment Approval Date". Adding this is contractionary to the reason why we did not directly link an amendment to a document. e.g. an amendment within the USDM is seen as a change to a study design version which may be reflected in a study protocol. 
- New description of appliesTo is still only referring to study sites. "... identifies the study site that applies to ..." Should also include the geographic scope or cohort.
- See point from Richard regarding StudyAmendment.changes relationship (class name should be changed in description)
- See point from Richard regarding StudyChange.changedSections: "DocumentContentReference classes" should probably be "DocumentContentReference class"
- StudyAmendmentImpact.description: to consider at next scrum => might be better to change this to text as it is more aligned to the other 'text' attributes in other classes.
- StudyAmendmentImpact.isSubstantial => it is about whether the described impact of the specified type is substantial. I think 'some' is to general.
- studyAmendmentImpact.type = Study Subject Rights : "The risk of impacting the rights of study subjects ...."
- studyAmendmentImpact.type = Study Data Reliability or Robustness. We decided to keep them combined. Although they are both different aspects from the data resulting from the study it is not expected that this would be addressed separately. Suggested text "The risk of impacting the data reliability or robustness of a clinical study"
- remove subjectEnrollment.type and SubjectEnrollment.code. They were inherited before from geographic scope. Keep SubjectEnrollment.quantity.
- remove substantialImpact from StudyAmendment

FYI @dih-cdisc @ASL-rmarshall

**Comment by BSnoeijerCD** [comment]
Decided during scrum:
Date types do not include what it is approving:
- Effective date
- Approval date
- Issued date

**Comment by ASL-rmarshall** [comment]
@EMuhlbradt @czwickl 
Looking at the alignment.csv file, it looks like there are a few things to be resolved:
- `substantialImpact` should be removed from `StudyAmendment`
- "relationship" should be changed to "Relationship" in column C for:
    - `notes` in `StudyAmendmentImpact`
    - `changedSections` in `StudyChange`
- `quantity` should not have been removed from `SubjectEnrollment` - it needs to be put back.

**Comment by EMuhlbradt** [comment]
> @EMuhlbradt @czwickl Looking at the alignment.csv file, it looks like there are a few things to be resolved:
> 
> * `substantialImpact` should be removed from `StudyAmendment`
> * "relationship" should be changed to "Relationship" in column C for:
>   
>   * `notes` in `StudyAmendmentImpact`
>   * `changedSections` in `StudyChange`
> * `quantity` should not have been removed from `SubjectEnrollment` - it needs to be put back.

Fixes made. Good catches!

**Comment by BSnoeijerCD** [comment]
@EMuhlbradt @czwickl 
All of the above is resolved. Two remaining issues
- The role of SubjectEnrollment.quantity is a "Complex Datatype Relationship" (also column C).
- I really would like to keep the reliability and robustness of the data combined as one impact type. Although these are two separate concepts. They are two sides of the same (data) coin. So they are often handled together. Having to split the impact in two separate items will therefore be forcing people to duplicate part of the information is and unwanted from a protocol design perspective I believe. Also the CDISC definition for Robustness is now missing.

**Comment by czwickl** [comment]
@BSnoeijerCD @daveih 
Regarding Berber's second bullet point above:

When Erin and I met to revert reliability and robustness data elements back into a single element, we noticed that for the impact on participant safety or rights, which is also a single question in the M11 template, we had split into two elements, 'safety' and 'rights', as well. It seems that 'reliability and robustness' should be treated the same as 'safety and rights'. Since we have a total of 4 concepts represented, we reasoned that is makes sense to split the two M11 questions (each containing 2 concepts), into separate data elements as we recalled Dave's comments that the USDM needs to support the ICH M11 document template, but that USDM goes beyond the M11 document. For that reason, we imagined a non-M11 use case where the 4 concepts would be treated separately and applied the general rule that data elements are easier to combine programmatically than to parse components from a single pre-combined element.

We have good definitions for all 4 concepts treated separately. If we lump them, we can only define the questions posed by the template as 'an M11 template data element containing/describing...', without actually defining what are meant by 'data reliability' vs 'data robustness'.

If the team could take one last look at this to make a final, final, final decision, taking into account the above, that would 'put the final nail in the coffin', so to speak. 

Thoughts anyone?
